cloudformation/compute-fleet-hit-substack.cfn.yaml

```diff
@@ -201,8 +201,8 @@ Resources:
               - Key: aws-parallelcluster-node-type
                 Value: Compute
     {%- for tag in tags %}
-              - Key: {{ tag.Key }}
-                Value: {{ tag.Value }}
+              - Key: "{{ tag.Key }}"
+                Value: "{{ tag.Value }}"
```

Comment:
do we have any validation on the chars allowed as part of a tag Key or Value? what happens if the tag key contains a double quote?